{{cookiecutter.repo_name|replace("_", "-")}}/Makefile

```diff
@@ -150,3 +150,26 @@ travis_docker_push: travis_docker_tag travis_docker_auth ## push to docker hub
 	docker push "openedx/{{cookiecutter.repo_name}}:$$TRAVIS_COMMIT"
 	docker push 'openedx/{{cookiecutter.repo_name}}:latest-newrelic'
 	docker push "openedx/{{cookiecutter.repo_name}}:$$TRAVIS_COMMIT-newrelic"
+
+# Docker dev commands below
```

Comment:
Should we also include a `restart` command, similar to `lms-restart` within devstack? Might be a convenience to avoid needing to run `docker-compose restart app`, for example.

Reply:
Sure, added